### PR TITLE
[IOTDB-3529] Rename Region to RegionGroup and optimize the SetTTL process

### PR DESCRIPTION
**Pull request by CRZbulabula**
1. Rename Region to RegionGroup in ConfigNode
2. Use AsyncDataNodeClientPool to send SetTTLReq

### CODE REVIEW

File: confignode/src/main/java/org/apache/iotdb/confignode/client/AsyncDataNodeClientPool.java [hunk 1]
```diff
@@ -283,6 +286,23 @@ public void flush(TEndPoint endPoint, TFlushReq flushReq, FlushHandler handler)
...
+   * @param endPoint The specific DataNode
+   */
+  public void setTTL(TEndPoint endPoint, TSetTTLReq setTTLReq, SetTTLHandler handler) {
+    for (int retry = 0; retry < 3; retry++) {
+      try {
+        clientManager.borrowClient(endPoint).setTTL(setTTLReq, handler);
+      } catch (IOException e) {
+        LOGGER.error("Can't connect to DataNode {}", endPoint, e);
+      } catch (TException e) {
+        LOGGER.error("Set TTL on DataNode {} failed", endPoint, e);
```
Comment (wangchao316) [inline confignode/src/main/java/org/apache/iotdb/confignode/client/AsyncDataNodeClientPool.java:301]: sleep...

File: confignode/src/main/java/org/apache/iotdb/confignode/client/handlers/SetTTLHandler.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      LOGGER.info("Successfully SetTTL on DataNode: {}", dataNodeLocation);
+    } else {
+      LOGGER.error("Failed to SetTTL on DataNode: {}, {}", dataNodeLocation, status);
+    }
+    latch.countDown();
+  }
+
+  @Override
+  public void onError(Exception e) {
+    LOGGER.error("Failed to SetTTL on DataNode: {}", dataNodeLocation);
```
Comment (RYH61) [inline confignode/src/main/java/org/apache/iotdb/confignode/client/handlers/SetTTLHandler.java:55]: latch.countDown();

File: confignode/src/main/java/org/apache/iotdb/confignode/client/AsyncDataNodeClientPool.java (hunk 1 above)
Comment (CRZbulabula) [reply confignode/src/main/java/org/apache/iotdb/confignode/client/AsyncDataNodeClientPool.java]: I've added a TODO here

File: confignode/src/main/java/org/apache/iotdb/confignode/client/handlers/SetTTLHandler.java (hunk 2 above)
Comment (CRZbulabula) [reply confignode/src/main/java/org/apache/iotdb/confignode/client/handlers/SetTTLHandler.java:56]: Fixed

**Review by RYH61 — APPROVED** [review]
LGTM~

**Review by qiaojialin — APPROVED** [review]

**Pull request merged**